dist/gcc-compatible/Hacl_Hash_SHA3.h

```diff
@@ -77,6 +77,8 @@ uint32_t Hacl_Hash_SHA3_hash_len(Hacl_Hash_SHA3_state_t *s);
 
 bool Hacl_Hash_SHA3_is_shake(Hacl_Hash_SHA3_state_t *s);
 
+void Hacl_Hash_SHA3_absorb_inner_32(uint32_t rateInBytes, uint8_t *b, uint64_t *s);
```

Comment:
right that's the only unpleasant bit from the C code change... if we wanted to fix it we could mark this function as private in Hacl.Impl.SHA3.Vec.fst and have Hacl.Hash.SHA3 `friend` Hacl.Impl.SHA3.Vec... not sure it's worth the trouble.

Reply:
coming back to this, I actually don't know why this ends up in the header... because the function definitely seems private so I would've expected it to end up in a private header (in `internal/`)... but let's fix this in a followup PR there's been enough happening in this one already